opencog/util/Counter.h

```diff
@@ -43,196 +43,196 @@ using boost::adaptors::map_values;
  * Given following pseudocode:
  * @code
  * for word in ['red', 'blue', 'red', 'green', 'blue', 'blue']:
- *   cnt[word] += 1
+ *	 cnt[word] += 1
```

Comment:
Oh, I get how you've dealt with that! You've performed a "    " -> "\t" replacement. But that's not good enough

1. doesn't align
2. create this sort of wrongs

A lot worse could happen too, like changing a string value, etc, leading to the kind of archetypal stories St Linas has just recounted.

Can't you have Neovim apply smart-tabs to all files in one go? I'm sure you can, if not, please migrate to the other church... ;-)

Reply:
The other option is to do a regex replacement, using `^` to indicate the start of a line, etc. But I think asking your editor to smart-tabs the whole thing is better as it will properly be aligned.

Reply:
Also, maybe create one commit per file (or h/cc pair), just in case one may want to bisect some problem, very unlikely IMO, just suggesting... Or if you prefer, only take care of Counter, it's up to you, just follow you're personal aesthetic inclinations.

Reply:
Well, I just used one shell command to retab all files using neovim. So 1 commit per file would be a lot more effort. 
And as I said everything is still aligned in my editor. So I guess I'll try again using the plugin you suggested. If that doesn't help then I'll just do Counter and leave the rest for future changes.